File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

# gets user input
try:
    h = int(get_int("Height: "))
except ValueError:
    h = int(get_int("Height: "))
if h > 8 or h < 1:
    h = int(get_int("Height: "))

# creates pyramid
for i in range(h):
    for j in range(h-(i+1)):
        print(" ", end="")
    print("#" * (i+1))

File: mario-less/mario.c
#include <cs50.h >
#include <stdio.h>

int main(void)
{
    //prompts user
    int height = get_int("Height: ");
    if (height <= 0 || height > 8)
    {
        height = get_int("Height: ");
    }

    //creates pyramid
    for (int i = 1; i <= height; i++)
    {
        //spaces
        for (int j = 0; j <  height - i; j++)
        {
            printf(" ");
        }
        //hashes
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

